ft232/libftdi.py

```diff
@@ -23,12 +23,17 @@
 from serial import (FIVEBITS, SIXBITS, SEVENBITS, EIGHTBITS, PARITY_NONE,
                     PARITY_EVEN, PARITY_ODD, STOPBITS_ONE, STOPBITS_TWO)
 
-for lib in ("libftdi.so", "libftdi.so.1", "libftdi.dylib", "libftdi.dylib.1", "libftdi1.dylib"):
+libftdi_filenames = ["libftdi.so", "libftdi.so.1", "libftdi.dylib", "libftdi.dylib.1", "libftdi1.dylib"]
+
+for lib in libftdi_filenames:
     try:
         ftdi = c.CDLL(lib)
         break
     except OSError:
         continue
+else:
+    print("Error: Unable to load libftdi!  Please make sure it is available as at least one of the following filenames:\n" + "\n".join(libftdi_filenames))
```

Comment:
Thanks! Can you just please drop this print as well? The exception should be more than enough and if a user wants to catch the exception and handle this in another way they probably don't want the extra noise.

Reply:
Alright, fixed.